Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/DemoVisualizer/Scripts/DrawOnTexture.cs

```diff
@@ -27,28 +27,44 @@ public class DrawOnTexture : MonoBehaviour
 
         public Material HeatmapOverlayMaterialTemplate;
 
-        private IMixedRealityInputSystem inputSystem = null;
+        private EyeTrackingTarget eyeTarget = null;
 
-        /// <summary>
-        /// The active instance of the input system.
-        /// </summary>
-        private IMixedRealityInputSystem InputSystem
+        private EyeTrackingTarget EyeTarget
         {
             get
             {
-                if (inputSystem == null)
+                if (eyeTarget == null)
                 {
-                    MixedRealityServiceRegistry.TryGetService<IMixedRealityInputSystem>(out inputSystem);
+                    eyeTarget = this.GetComponent<EyeTrackingTarget>();
                 }
-                return inputSystem;
+                return eyeTarget;
             }
         }
-
+        
         private void Update()
         {
-            if (UseLiveInputStream && InputSystem?.EyeGazeProvider?.GazeTarget == gameObject && InputSystem.EyeGazeProvider.IsEyeGazeValid)
+            //if (UseLiveInputStream && InputSystem?.EyeGazeProvider?.GazeTarget == gameObject && InputSystem.EyeGazeProvider.IsEyeGazeValid)
```

Comment:
Was this supposed to be uncommented? Or should the Update() function get removed (i..e because it's all commented out)

Reply:
Good catch - will check!

Reply:
Tested it and yes, this is not required anymore. Move the update to OnLookAt().